internal/command/generate.go

```diff
@@ -296,6 +378,21 @@ arguments.
 	},
 }
 
+func parseResourceUid(raw string) framework.ResourceUid {
+	parts := strings.SplitN(raw, "#", 2)
+	firstParts := strings.SplitN(parts[0], ".", 2)
+	secondParts := strings.SplitN(parts[1], ".", 2)
+	resType := firstParts[0]
+	var resClass *string
```

Comment:
Shouldn't this be `default` as default value :) ?

Reply:
The NewResourceUid thing already handles that for us here 👍

Reply:
Cool sorry, didn't check this was happening!